.github/workflows/ccpp.yml

```diff
@@ -0,0 +1,15 @@
+name: C/C++ CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: mkdir
+      run: mkdir build && cd build && cmake ..
+    - name: make
+      run: make
```

Comment:
`make -C build` technically

Reply:
I forgot this was opened against my master, probably should back this out